doc/en/user/source/services/wms/global.rst

```diff
@@ -25,3 +25,13 @@ ENABLE_JSONP
 
 Can be ``true`` or ``false`` (defaults to: ``false``).
 When ``true`` the JSONP (text/javascript) output format is enabled.
+
+GETMAPKVPREQUESTREADER_TIMEOUT
+-------------
+
+Sets the max connection timeout time in milliseconds, to connect or await new socket packets in the KVP request reader.  The default is 30000.
+
+GETMAPKVPREQUESTREADER_MAX_REQUEST_TIME
```

Comment:
I would try to improve the name, for example ``GET_MAP_KVP_REQUEST_READER_TIMEOUT_MS``, anyway, the name seems to imply that is a time-out specific WMS GetMAP requests encoded as KVP, but no the description? What about WMS requests encoded as POST (GeoServer supports this although this is not standard).